Sources_v3/WebSocketClient/WebSocketClient_Tests.swift

```diff
@@ -245,7 +245,7 @@ class WebSocketClient_Tests: StressTestCase {
     
     func test_webSocketPingController_connectionStateDidChange_calledWhenConnectionChanges() {
         test_connectionFlow()
-        AssertAsync.willBeEqual(pingController.connectionStateDidChange_connectionStates,
+        AssertAsync.willBeEqual(pingController.connectionStates,
```

Comment:
These variable names follow specific conventions. It's the `<name of the function>_<recorded value>`. Please don't randomly change this.

Reply:
It was really long name, more then 40 characters and I think it's clear this is a collection connection states.